HtmlCode/style.css

```diff
@@ -0,0 +1,20 @@
+*{
+    margin: 0px;
+}
+#container{
```

Comment:
Appreciate what you did with css, rendering different color background in all 3 cases.

Reply:
> Appreciate what you did with css, rendering different color background in all 3 cases.
Thank you @Swati27